### ISSUE TITLE
Model requirments 

### ISSUE THREAD

**Issue by ayanasser** [mchengny/RWF2000-Video-Database-for-Violence-Detection#9]
Hello, 
Thank you for your effort, 
I have used your keras model but i am facing some problems that related to cuda and cudnn 
so could u give me the versions of the following packages ?
TF
Keras
Cuda
CuDnn

and could u have evaluation metric that describe that model accuracy or anything

**Comment by mchengny** [comment]
> Hello,
> Thank you for your effort,
> I have used your keras model but i am facing some problems that related to cuda and cudnn
> so could u give me the versions of the following packages ?
> TF
> Keras
> Cuda
> CuDnn
> 
> and could u have evaluation metric that describe that model accuracy or anything

Hi, FYR:https: //github.com/mchengny/RWF2000-Video-Database-for-Violence-Detection/issues/6

**Comment by ayanasser** [comment]
okai, i ran it successfully and  i will post the requirements i got, 
but i have another question, this is the output i got
pred= [[0.76591665 0.23408334]]
Dose the pred[0] is the violence indication or not ?

**Comment by ayanasser** [comment]
The requirements i got are:
cuda:10.0-cudnn7
Keras (2.3.1)
tensorflow-gpu (1.13.1)

**Comment by mchengny** [comment]
> The requirements i got are:
> cuda:10.0-cudnn7
> Keras (2.3.1)
> tensorflow-gpu (1.13.1)

Hi，Thanks for your efforts that others can get a reference.

**Comment by mchengny** [comment]
> @mchengny
> okai, i ran it successfully and i will post the requirements i got,
> but i have another question, this is the output i got
> pred= [[0.76591665 0.23408334]]
> Dose the pred[0] is the violence indication or not ?

HI, you could check the printed information after initializing the data generator. If "violent" is assigned to class 0, then pred[0] means  "violent". Otherwise, pred[0] means "non-violent".

**Comment by gezhaoDL** [comment]
我用的tensorflow镜像，tensorflow/tensorflow:1.13.1-gpu-py3，keras安装的2.3.1，但是执行到model = load_model('./Models/keras_model.h5')时候报错Segmentation fault (core dumped)
 请问可能是什么原因？

**Comment by baigang666** [comment]
> HI, you could check the printed information after initializing the data generator. If "violent" is assigned to class 0, then pred[0] means "violent". Otherwise, pred[0] means "non-violent"

我没有理解您说的。我现在使用的是预训练模型，我从那里确认pred[0]代表“暴力”，还是代表“非暴力”
谢谢
